web_external/Datasets/ApplyMetadataView.js

```diff
@@ -0,0 +1,261 @@
+// Model for a metadata file
+isic.models.MetadataFileModel = girder.Model.extend({
+    name: function () {
+        var time = girder.formatDate(this.get('time'), girder.DATE_SECOND);
+        var fileName = this.get('file').name();
+        var userName = this.get('user').name();
+        return time + ' \u2014 ' + fileName + ' \u2014 ' + userName;
```

Comment:
Em dash is much wider than a hyphen, and the role of the character here is to be a helpful separator between the components. A more advanced user interface here than a select box could be appropriate, and that would avoid using characters to separate information.